### PR TITLE
response

### PR DESCRIPTION
**Pull request by jsmeng324**

### CODE REVIEW

File: lib/challenge.js [hunk 1]
```diff
@@ -19,6 +19,17 @@
   // - park
 
 // Answer Starts Here
+  'calendar'
+  Description: a calendar app that keep track of things for users
+  Every calendar has:
+    - a user
+    - a date
+    - a time
+    - an event
```
Comment (GA-MEB) [inline lib/challenge.js:28]: Only one?

File: lib/challenge.js [hunk 2]
```diff
@@ -19,6 +19,17 @@
...
+  Every calendar can:
+    - add/delete new date/time/event
+    - set reminder before event date/time
+    - invite other user to certain events
```
Comment (GA-MEB) [inline lib/challenge.js:32]: This abstraction is pretty good - can you think of two others?

File: lib/challenge.js [hunk 3]
```diff
@@ -51,7 +65,16 @@ var pet_owner;
...
 // Answer Starts Here
-
+some_pet = {
+    name: "zoe",
+    species: "dog",
+    breed: "husky",
+    noise: "bark",
+
+    makeNoise: function(){
+      return this.name + " the " + this.species + " says " + this.noise + "!";
```
Comment (GA-MEB) [inline lib/challenge.js:75]: Nice!

**Comment by GA-MEB** [comment]
3.5 of 4

I took off 0.5 from Q1, but I could be persuaded to add it back if you can demonstrate two other good abstractions for that problem.

Overall, very nice work!

File: lib/challenge.js (hunk 2 above)
Comment (jsmeng324) [inline lib/challenge.js:32]: Completely misunderstood the question... thanks for giving me the second chance, Matt. Here are the two others I can think of.
(1) 
Description: a calendar app that help people host local events
Every calendar has:
    - host name
    - partners name
    - location
    - date/time
    - number of RSVP
Every calendar can:
    - add/delete/modify partners
    - modify location & date/time
    - change number of people RSVP

(2) 
Description: a calendar app that helps show runner to keep track of show information
Every calendar has:
    - name of the show
    - performer's name
    - location & date/time
    - maximum number of seats
    - ticket price
Every calendar can:
    - add/delete show info (including name, date, time, location, etc)
    - check available seats
    - calculate revenue
    - calculate attendance

**Pull request closed without merging**